src/onboarding/steps.ts

```diff
@@ -0,0 +1,208 @@
+import { BIOMETRY_TYPE } from 'react-native-keychain'
+import { initializeAccount } from 'src/account/actions'
+import {
+  choseToRestoreAccountSelector,
+  recoveringFromStoreWipeSelector,
+} from 'src/account/selectors'
+import {
+  numberVerifiedCentrallySelector,
+  skipVerificationSelector,
+  supportedBiometryTypeSelector,
+} from 'src/app/selectors'
+import { setHasSeenVerificationNux } from 'src/identity/actions'
+import * as NavigationService from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+import { store } from 'src/redux/store'
+
+export const FIRST_ONBOARDING_SCREEN = Screens.NameAndPicture
+export const END_OF_ONBOARDING_SCREEN = Screens.WalletHome
```

Comment:
this might be a premature optimization, but in a future where we want to choose the first/last onboarding screen by the user's A/B test cohort, it would be more convenient to have these wrapped by getter functions, like `getFirstOnboardingScreen()`

Reply:
they can still just return a constant value for now. just thinking if they're exported and used elsewhere, it might be better to wrap them in getter functions